theme/themes/default/collections/table.variables

```diff
@@ -152,7 +152,7 @@
 @attachedTopOffset: 0px;
 @attachedBottomOffset: 0px;
 @attachedHorizontalOffset: -@borderWidth;
-@attachedWidth: ~"calc(100% + "-@attachedHorizontalOffset * 2~")";
+@attachedWidth: calc(100% - (@attachedHorizontalOffset * 2));
```

Comment:
maybe you should keep the `~" "` around https://github.com/less/less.js/issues/974#issuecomment-9229470